tests/Repository.py

```diff
@@ -1296,11 +1296,11 @@ def testGetSubscribers(self):
 
     def testCreatePull(self):
         pull = self.repo.create_pull(
-            "Pull request created by PyGithub",
-            "Body of the pull request",
-            "topic/RewriteWithGeneratedCode",
-            "BeaverSoftware:master",
-            True,
+            title="Pull request created by PyGithub",
+            body="Body of the pull request",
+            base="topic/RewriteWithGeneratedCode",
+            head="BeaverSoftware:master",
+            maintainer_can_modify=True,
```

Comment:
with
```suggestion
            draft=False,
            maintainer_can_modify=True,
```
there would be no need to change the replay data, and we would test that attribute

Reply:
good idea